kemp-loadmaster-multi-nic/azuredeploy.json

```diff
@@ -0,0 +1,230 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vmName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the VM"
+            }
+        },
+        "balPassword": {
+         "type": "securestring",
+          "metadata": {
+          "description": "Password for bal account"
+        }
+       },
+        "vmSize": {
```

Comment:
any reason this can't be a variable - many of those sizes are not available in all regions/clouds